packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -309,6 +311,20 @@ class SemanticsData extends Diagnosticable {
   ///  * [UiKitView], which is the platform view for iOS.
   final int platformViewId;
 
+  /// The maximum number of characters that can be entered into an editable
```

Comment:
We should specify exactly what we mean by "characters" here

Reply:
Added clarification.